packages/app/src/components/nominationInfo/healthProviderDetail.js

```diff
@@ -1,69 +1,51 @@
 import React, { useContext } from 'react';
-import styles from "./styles.module.css";
-import { Link, useParams } from "react-router-dom";
-import { SearchResultDataContext } from '../../utils/context/SearchResultsContext';
-import { NominationsDataContext } from '../../utils/context/NominationsContext';
-
+import styles from './styles.module.css';
+import { useParams } from 'react-router-dom';
 
 function HealthProviderDetail(props) {
-      
-    const [SearchResultData, setSearchResultData] = useContext(
-      SearchResultDataContext
-      );
-      
-    const [NominationsData, setNominationsData] = useContext(
-      NominationsDataContext
-      );
-        
-    const findSearchResults = (searchTerm) => {
-          let filteredNoms = []
-          if(NominationsData && searchTerm !== undefined) {
-            filteredNoms = NominationsData.filter((nomination) => {
-              return [
-                nomination.providerName,
-                nomination.patientName,
-                nomination.nominationName,
-                nomination.representativeName,
-              ].some((nom) => nom.includes(searchTerm));
-            });
-            setSearchResultData(filteredNoms);
-          } 
-        }
+  function openNewBackgroundTab(val) {
+    var a = document.createElement('a');
+    a.href = `/searchhealthprovider/${val}`;
+    var evt = document.createEvent('MouseEvents');
+    evt.initMouseEvent('click', true, true, window, 0, 0, 0, 0, 0, true, false, false, false, 0, null);
```

Comment:
InitMouseEvent is deprecated and not really supported any more: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent We should try to use a newer browser API for this.

Reply:
I reworked this using window.open(). Hopefully a simpler solution.  From what I can tell this is working now.